compiler/rustc_const_eval/src/const_eval/eval_queries.rs

```diff
@@ -43,6 +43,7 @@ fn eval_body_using_ecx<'tcx, R: InterpretationResult<'tcx>>(
                     | DefKind::AnonConst
                     | DefKind::InlineConst
                     | DefKind::AssocConst
+                    | DefKind::Ctor(_, CtorKind::Const)
```

Comment:
since this is all just feature gated now can u remove this change and we can figure out how to handle these constructors at a later time so we don't have to worry about it rn